server/game/cards/16-DT2/GhostlyCommunion2.js

```diff
@@ -37,7 +37,7 @@ class GhostlyCommunion2 extends SpellCard {
             condition: () => this.game.shootout &&
                 (this.game.shootout.shootoutLocation.locationCard.hasKeyword('holy ground') || 
                 this.game.shootout.shootoutLocation.locationCard.adjacentLocations().some(location => location.locationCard.hasKeyword('holy ground')) ||
-                this.game.shootout.shootoutLocation.locationCard.isNearby(this.controller.outfit.uuid)),
+                this.game.shootout.shootoutLocation.locationCard.isNearby(this.controller.outfit.uuid)) || this.game.shootout.shootoutLocation === this.game.townsquare.uuid,
```

Comment:
the shootout location's location card has to be used for comparison. 
and the conditional should be inside the parenthesis with the other  shootout location checks.

```suggestion
                this.game.shootout.shootoutLocation.locationCard.isNearby(this.controller.outfit.uuid) ||
                this.game.shootout.shootoutLocation.locationCard.uuid === this.game.townsquare.uuid
                ),
```

Reply:
You do not need to get the `locationCard` of the `shootoutLocation`. The class `GameLocation` (class of shootoutLocation) has an `uuid` getter, so you can just do this:
```
this.game.shootout.shootoutLocation.uuid === this.game.townsquare.uuid
```